app/controllers/item.js

```diff
@@ -127,6 +128,15 @@ export default Ember.Controller.extend({
       );
     },
 
+    viewChildCategory(category) {
```

Comment:
shouldn't this be `viewParentCategory` ? since it's navigating to `category.get("parentId")` ? 🤔 

(maybe that function could be shared with the browse controller somehow, they look pretty similar btw)

Reply:
yeah, it is navigating to parent category but, here child category is preselected. on clicking any sub category from `browse` page @patrixr .
Maybe we can rename it to `setChildCategoryAndRedirect`
Let me know if it sounds good !!